utils/stellar.js

```diff
@@ -60,6 +61,68 @@ exports.stellar = (function() {
     })
   }
 
+  createEscrowAccount = (secretKey, shaSignerKey, callback) => {
+    StellarSdk.Network.useTestNetwork();
+    let sourceKeypair = StellarSdk.Keypair.fromSecret(secretKey);
+    let escrowKeypair = StellarSdk.Keypair.random();
+    
+    (async () => {
+      try{
+        
+        const account = await stellarServer.loadAccount(sourceKeypair.publicKey())
+        let transaction = new StellarSdk.TransactionBuilder(account)
+        .addOperation(StellarSdk.Operation.createAccount({
+          destination: escrowKeypair.publicKey(),
+          startingBalance: '100'
+        })).build();
+        transaction.sign(sourceKeypair);
+        stellarServer.submitTransaction(transaction).then(() => {
+          return stellarServer.loadAccount(escrowKeypair.publicKey())
+        }).then((escrowAccount) => {
+          let transaction = new StellarSdk.TransactionBuilder(escrowAccount)
+          .addOperation(StellarSdk.Operation.setOptions({
+            signer: {sha256Hash: shaSignerKey, weight: 2},
+            masterWeight: 4,
```

Comment:
Some of these numbers seem a bit "magic" to me, in that it's hard to tell why the keys hold their specific values. For example, using `BASELINE_REDUNDANCY` to stand in for `3` in the method which creates shards duplicates helps you understand why the value is 3 a bit. Do you think that adding them as constants might help that here?

Reply:
I will be modifying these weights a bit, but they're not magic numbers, they're to manipulate the amount of power the data host has over the escrow account. On

Reply:
Yeah, definitely didn't mean those weights were random - just that I couldn't tell why `masterWeight` was `4` and not `24`, for example. Maybe a comment or link to documentation is more appropriate here, though. I'll leave that up to you.